fix(server-admin-ui): Use mark in place of mark+text in minimized sidebar

bkeepers opened the pull request:
This is a small tweak to use just the mark in place of mark+text in minimized sidebar. It also slightly decreases the size of the logo when expanded to improve the alignment with the nav.

<table>
<tr>
<th>Before</th>
<td><img width="201" alt="image" src="https://github.com/user-attachments/assets/9e2c1bed-36fa-4376-a07d-12b2a73830c8" /></td>
<td><img width="174" alt="image" src="https://github.com/user-attachments/assets/ffc8ada1-8d89-4ae0-b8ae-0854fd9f3a7e" /></td>
</tr>
<tr>
<th>After</th>
<td>
<img width="200" alt="image" src="https://github.com/user-attachments/assets/d86cf395-d280-4332-a39c-816f08ceeae4" />
</td>
<td><img width="166" alt="image" src="https://github.com/user-attachments/assets/96ad0fd8-f393-4358-a3ea-873cc0b1ffc6" /></td>
</tr>
</table>

sbender9 commented:
Is going to mess with vendors who replace the logo?

bkeepers commented:
It only changes the default values for the existing CSS variables, so it shouldn't. Are there any docs for vendors that I can read to make sure I'm not missing something?

sbender9 commented:
They can provided a different image by placing a file 'logo.svg' in the ~/.signalk directory.

bkeepers commented:
> They can provided a different image by placing a file 'logo.svg' in the ~/.signalk directory.

Ah, ok. I just pushed an update that will server the same logo for both paths. I could also update it to check for a different file (`logo-square.svg`?) and serve that if it exists.

For now it works the same as previously:

<img width="201" alt="image" src="https://github.com/user-attachments/assets/f064ebb5-8d90-4f27-a75a-3f9b5863cee5" />

<img width="187" alt="image" src="https://github.com/user-attachments/assets/dc64e398-5789-4b57-a64c-4bbc608bd697" />

tkurki commented:
Yes, I think we should parity when using the custom logo. Maybe `logo-image.svg`? The exact name is not crucial, as we already broke with the original naming with just `logo.svg`.

bkeepers commented:
I updated this to look for `logo-minimized.svg` and added a note in the README where the `logo.svg` feature was documented. This should be good to go.

tkurki commented:
Build is failing with a styling issue.

bkeepers commented:
> Build is failing with a styling issue.

Sorry about that, fixed now.
